modules/async-icq/testing/previous_images/README.md

```diff
@@ -2,8 +2,8 @@
 
 This section is used for upgrades e2e test.
 
-v7.0.0
+v8.0.0
 
 - <https://github.com/cosmos/ibc-apps/commit/e5a274cf6fc2eb965a9f8da4bdeb7c718d06661d>
-- docker build . -t icq-host:v7.0.0 -f Dockerfile.icq
-- docker save icq-host:v7.0.0 > icq-host_7_0_0.tar
+- docker build . -t icq-host:v8.0.0 -f Dockerfile.icq
+- docker save icq-host:v8.0.0 > icq-host_8_0_0.tar
```

Comment:
This commit is v7.1.1. Also ref above comments about v7.1.1 as the baseChain for upgrades